src/components/ValidatedInput/ValidFieldTextArea.tsx

```diff
@@ -14,47 +14,36 @@ import "./ValidFieldTextArea.scss";
 import { FieldCounter, FieldErrorMessage, FieldHelperText, FieldLabel } from "../Form/utility";
 
 export interface ValidFieldTextAreaProps extends FieldTextAreaStatelessProps {
-  label?: string;
   value: string;
   errMessage?: string;
   isInvalid?: boolean;
```

Comment:
do we still need the `isInvalid` props? Since the other component has been deprecated

Reply:
removed